docs/www/quick-start-docker.md

```diff
@@ -72,7 +72,7 @@ docker run -d \
 --hostname=redpanda-1 \
 --net=redpandanet \
 -p 8082:8082 \
--p 9092:9092 \
+-p 9092:9092 -p 9644:9644 \
```

Comment:
I think for consistency it may be worth it to move this to a new line? (same in every other place)

Reply:
agreed